### PR TITLE
Custom calendar settings for first day of week and default view

### PR DESCRIPTION
**Pull request by sbonaime**

### CODE REVIEW

**Comment by sbonaime** [comment]
I don't know how to set default values for 'calendar_view' and 'calendar_first_day_of_week'

File: NEMO/templates/customizations.html [hunk 1]
```diff
@@ -5,6 +5,38 @@
 		<h1>Customizations</h1>
 		<p>You can customize portions of NEMO to suit your organization's needs.</p>
 
+		<div class="panel panel-default">
```
Comment (mrampant-nist) [inline NEMO/templates/customizations.html:8]: can you move this right after the email addresses?

**Comment by mrampant-nist** [comment]
this is great, thank you! I added a few small requests for changes, but this is really nice and works really well

**Comment by sbonaime** [comment]
> this is great, thank you! I added a few small requests for changes, but this is really nice and works really well

by the way, we can think about others parameters like :
- scrollTime,  The hour the day starts
- date format

**Comment by mrampant-nist** [comment]
absolutely. it will be easy to add more from what you have already done

File: NEMO/templates/customizations.html [hunk 2]
```diff
@@ -5,6 +5,38 @@
...
+									<label><input type="radio" id="calendar_view" name="calendar_view" {% if calendar_view == 'month' %}checked{% endif %} value="month"> Month view</label>
+									<label><input type="radio" id="calendar_view" name="calendar_view" {% if calendar_view == 'agendaDay' %}checked{% endif %} value="agendaDay"> Day view</label>
+								</div>
+							</div>
+						</div>
+						<div class="form-group">
+							<div class="col-md-12">
+								<label class="control-label col-md-2" >First day of the week </label>
+								<div class="checkbox">
+									<label><input type="radio" id="calendar_first_day_of_week" name="calendar_first_day_of_week" {% if calendar_first_day_of_week == '1' %}checked{% endif %} value="1"> Monday </label>
```
Comment (mrampant-nist) [inline NEMO/templates/customizations.html:27]:
```suggestion
									<label><input type="radio" id="calendar_first_day_of_week" name="calendar_first_day_of_week" {% if not calendar_first_day_of_week or calendar_first_day_of_week == '1' %}checked{% endif %} value="1"> Monday </label>
```

File: NEMO/templates/customizations.html [hunk 3]
```diff
@@ -5,6 +5,38 @@
...
+		<div class="panel panel-default">
+			<div class="panel-body">
+				<h3>Calendar settings</h3>
+					<form method="POST" action="{% url 'customize' 'calendar_settings' %}" class="form-horizontal">
+						{% csrf_token %}
+						<div class="form-group">
+							<div class="col-md-12">
+								<label class="control-label col-md-2" >Default view</label>
+								<div class="checkbox">
+									<label><input type="radio" id="calendar_view" name="calendar_view" {% if calendar_view == 'agendaWeek' %}checked{% endif %} value="agendaWeek"> Week view</label>
```
Comment (mrampant-nist) [inline NEMO/templates/customizations.html:17]:
```suggestion
									<label><input type="radio" id="calendar_view" name="calendar_view" {% if not calendar_view or calendar_view == 'agendaWeek' %}checked{% endif %} value="agendaWeek"> Week view</label>
```

**Comment by sbonaime** [comment]
> absolutely. it will be easy to add more from what you have already done

I'll go for date format

File: NEMO/views/calendar.py [hunk 4]
```diff
@@ -53,10 +53,23 @@ def calendar(request, tool_id=None):
 
 	tools = Tool.objects.filter(visible=True).order_by('_category', 'name')
 	rendered_tool_tree_html = ToolTree().render(None, {'tools': tools, 'user': request.user})
+	
+	# default value for calendar_view is 'agendaWeek'
+	calendar_view = get_customization('calendar_view')
+	if calendar_view == '':
```
Comment (mrampant-nist) [inline NEMO/views/calendar.py:59]:
```suggestion
	if not calendar_view:
```

File: NEMO/views/calendar.py [hunk 5]
```diff
@@ -53,10 +53,23 @@ def calendar(request, tool_id=None):
...
+		calendar_view = 'agendaWeek'
+	
+	# default value for calendar_first_day_of_week is 1 (Monday) 
+	calendar_first_day_of_week = get_customization('calendar_first_day_of_week')
+	if calendar_first_day_of_week == '':
```
Comment (mrampant-nist) [inline NEMO/views/calendar.py:64]:
```suggestion
	if not calendar_first_day_of_week:
```

**Comment by mrampant-nist** [comment]
I am going to merge this one, and you can submit a new PR for date format if you'd like. I suspect it's going to be a bit more involved.

**Pull request merged**